### DISCUSSION TITLE
Elementary as a Desktop Environment?

### DISCUSSION THREAD

**Discussion by bkfirmen** [elementary/triage#392] [Ideas]
I don't know if there was a discussion about this already. But while I was thinking about elementaryOS, I realized (as a noob) that the entirety of the ideas and goals of elementaryOS (user-friendliness mostly) could be realized by just providing the pantheon Desktop-Environment.
This would save a lot of Developer Resources and could mean faster developement for the pantheon desktop. And the DE could be implemented in f.e. Linux Mint (maybe make a deal with them so they don't modify the DE), which seems to be the nearest to elementaryOS. But other distros could obviously also profit.
I'm probably wrong, or maybe this was already discussed (and I wasn't able to find it) then correct me. It's just an idea, and I'm interested in your thoughts :)

**Comment by danirabbit** [comment]
Pantheon is already available for any other distro to package up and is available in Fedora and NixOS and I think used by default in a couple of other distros.

It is definitely a lot *more* work to make pantheon work on any distro than it is to make it work on our own distro, but we do our best with the help of our downstreams and community :)

**Comment by bkfirmen** [comment]
Ah, really cool to hear, that pantheon is also on other distributions. But is it really necessary to have an own distro for it? I mean, the team could focus on just implementing it in one distro (to safe dev work).

**Reply by danirabbit** [reply]
It's imo not significantly more work than the value we get in return. There's really not a huge savings to be had by not producing a distribution but discontinuing it would eliminate nearly all of our revenue, we'd lose control of our updates cycle, we wouldn't be able to provide distro patches, we'd be at the mercy of a distro for defaults and branding, it's unclear what would happen to AppCenter etc. It would be a significant loss

**Comment by bkfirmen** [comment]
Ok, yes, that sounds logical. I will close this now. 
But thanks for your time :)
